### PR TITLE
adding message support function to klaudsol website

### PR DESCRIPTION
**Pull request by angeli-mercedes**
- added a message support function to the send message button found at the contact us form. 
- sender details and written message will automatically be sent to Slack (klaudsol-message-support channel)

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import Script from 'next/script';
+import ContactUsForm from './contact_us';
 
 export default function Home() {
   return (
     <>
       <Head>
         {/*<!-- Site title -->*/}
-        <title>KlaudSol - Cloud-based Business Apps</title>
+        <title>Klaudsol - Cloud-based Business Apps</title>
```
Comment (ardeearam) [inline pages/index.js:11]: The proper spelling is `KlaudSol` - with a capital S. It stands roughly for "Cloud Solutions".

File: pages/contact_us.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    async function sendMessage(e) {
+       e.preventDefault();
+    
+       var today= new Date();
+       var date_sent = today.toLocaleString()
+       const data = {
+        "text": `----- KLAUDSOL MESSAGE SUPPORT ----- \nDATE SENT: ${date_sent} \n\nSENDER DETAILS: \nName: ${name} \nEmail: ${email} \nSubject: ${subject} \n\nMessage: ${message}`,
+       }
+
+       let result = await axios.post(`https://${process.env.NEXT_PUBLIC_WEBHOOK_HOST_NAME}/${process.env.NEXT_PUBLIC_WEBHOOK_PATH}/${process.env.NEXT_PUBLIC_WEBHOOK_LAST_QUERY_STRING}`, JSON.stringify(data), {
```
Comment (ardeearam) [inline pages/contact_us.js:21]: I think there is no need to separate the URL into three parts? When you generate a webhook link from Slack anyway, it generates the URL in full.

**Review by ardeearam — CHANGES_REQUESTED** [review]
Kindly address code comments. Thank you!

File: pages/contact_us.js (hunk 2 above)
Comment (angeli-mercedes) [reply pages/contact_us.js:21]: This is noted po sir. I will update the code po right away. Thank you po!

**Review by ardeearam — APPROVED** [review]

**Comment by ardeearam** [comment]
Looks good to me. Let's merge this tomorrow, while I show you and give you access to AWS Amplify.

In the meantime, you can either pick up a new task from the board, or you can help Julianne with her portfolio task.

**Comment by geli-beans** [comment]
Noted po, sir! Thank you very much po.

**Pull request merged**